src/Camelot.Services.Windows/WindowsShellLinksService.cs

```diff
@@ -0,0 +1,30 @@
+using System;
+using System.IO;
+using System.Runtime.Versioning;
+using Camelot.Services.Abstractions;
+using Camelot.Services.Windows.ShellIcons;
+
+namespace Camelot.Services.Windows;
+
+[SupportedOSPlatform("windows")]
+public class WindowsShellLinksService : IShellLinksService
+{
+    public bool IsShellLink(string path)
+    {
+        if (string.IsNullOrEmpty(path)) 
+            return false;
+        var ext = Path.GetExtension(path).ToLower();
```

Comment:
Please use `IPathService` instead of `Path`

Reply:
will do.